suite/suite_test.go

```diff
@@ -350,7 +350,7 @@ func TestRunSuite(t *testing.T) {
 type SuiteSetupSkipTester struct {
 	Suite
 
-	setUp bool
+	setUp    bool
```

Comment:
these changes are not related to imports

Reply:
goimports does both import ordering as well as calling gofmt.

Reply:
Hmm. This change would lead me to believe the go formatting check (defined during CI)[https://github.com/stretchr/testify/blob/master/.travis.gofmt.sh#L5) is not being run.

FYI: @georgelesica-wf

Reply:
Agreed, weird...

Reply:
Fixed, hopefully... https://github.com/stretchr/testify/pull/804